clang-tools-extra/clangd/CodeComplete.h

```diff
@@ -52,6 +52,9 @@ struct CodeCompleteOptions {
   /// For example, private members are usually inaccessible.
   bool IncludeIneligibleResults = false;
 
+  /// Whether the experimental modules support are enabled.
+  bool ExperimentalModulesSupport = false;
```

Comment:
can you rename this to `ForceLoadPreamble`? with a comment explaining what it does

Reply:
Done